website/siteConfig.js

```diff
@@ -153,6 +153,7 @@ const siteConfig = {
     return `
       <!DOCTYPE html>
       <link rel="stylesheet" media="screen" href="https://fontlibrary.org/face/hk-grotesk" type="text/css"/>
+      <link rel="stylesheet" media="screen" href="https://fontlibrary.org/face/hkgrotesklight" type="text/css">
```

Comment:
Not sure if we need this, is this font used somewhere?

Reply:
Yes it is used [here](https://github.com/aragon/hack/blob/66f6e030fc6ca1a09ac53fd9f71edce98be72c49/website/static/css/custom.css#L44) and on other places as well. I wasn't aware of this link missing cause I had the font installed on my fontbook.